google/cloud/bigquery/client.py

```diff
@@ -2319,17 +2323,19 @@ def extract_table(
             timeout (Optional[float]):
                 The number of seconds to wait for the underlying HTTP transport
                 before using ``retry``.
-        Args:
-            source (google.cloud.bigquery.table.TableReference): table to be extracted.
-
+            source_type (str):
+                (Optional) Type of source to be extracted.``Table`` or ``Model``.
```

Comment:
Yes, they are very inconsistent in that matter as in same method they used two various type for optional keyword, I checked whole file and found it only differ in case of time out so may be in different PR we can make consistent and change the declaration type of timeout parameter.

Reply:
:+1:  That would be a good idea!